File: SingleAPIExample/CSharp/Approve.cs
using Splitit.SDK.Client.Api;
using Splitit.SDK.Client.Model;

namespace Examples
{
    public class ApproveExample
    {
        InstallmentPlanApi PlanApi { get; }

        public void Approve(string installmentPlanNumber)
        {
            var request = new ApproveInstallmentPlanRequest
            {
                InstallmentPlanNumber = installmentPlanNumber,
                AreTermsAndConditionsApproved = true,
            };

            var approveResponse = PlanApi.InstallmentPlanApprove(request);
        }
    }
}

File: SingleAPIExample/PHP/Approve.php
<?php

$sessionId = '{{SessionID}}';
\SplititSdkClient\Configuration::sandbox()->setApiKey('{{APIKey}}');

$apiInstance = new SplititSdkClient\Api\InstallmentPlanApi(\SplititSdkClient\Configuration::sandbox(), $sessionId);

$request = new \SplititSdkClient\Model\ApproveInstallmentPlanRequest([
    'installment_plan_number' => '{{InstallmentPlanNumber}}'
]);

try {
    $result = $apiInstance->installmentPlanApprove($request);
    print_r($result);
} catch (Exception $e) {
    echo 'Exception when calling InstallmentPlanApi->installmentPlanApprove: ', $e->getMessage(), PHP_EOL;
}
